packages/boxel-motion/addon/utils/measurement.ts

```diff
@@ -19,10 +19,12 @@ function runWithoutAnimations(playAnimations: boolean) {
     let currentTimes: number[] = [];
     animations.forEach((a) => {
       a.pause();
-      currentTimes.push(a.currentTime || 0);
+      currentTimes.push((a.currentTime as number) || 0);
```

Comment:
These castings are all because of [this error pattern](https://github.com/cardstack/boxel/actions/runs/5764305800/job/15627994396#step:7:28):

```
Argument of type 'CSSNumberish' is not assignable to parameter of type 'number'.
```

Is this a valid casting? 🤔

Reply:
It's weird that they wrote the types that way. The WHATWG spec for this says it's definitely what typescript would call `number`. So this cast is safe.